src/components/_classes/nested/NestedComponent.js

```diff
@@ -927,22 +927,24 @@ export default class NestedComponent extends Field {
 
   setNestedValue(component, value, flags = {}) {
     component._data = this.componentContext(component);
+    let componentFlags = {...flags};
```

Comment:
Is this necessary anymore now that we are no longer mutating the "dirty" property?  I would prefer to not have to clone if we don't need to.